### PR TITLE
Travis and Appraisal changes

### PR DESCRIPTION
**Pull request by kbrock**
- Added definitions for active record 5.2 and 6.0
- Upgraded gemfiles to be more similar to other projects.
- Alphabetized the list of services and gems
- updated the versions of dependent gems

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,26 @@
 ---
+sudo: false
 language: ruby
 cache: bundler
 rvm:
-- 2.4.1
-- 2.5.3
+- 2.4.6
```
Comment (NickLaMuro) [inline .travis.yml]: > test against latest versions of gems 

... and rubies :trollface:

File: .travis.yml [hunk 2]
```diff
@@ -13,14 +13,14 @@ env:
 - DB=pg
 - DB=sqlite3
 gemfile:
-- gemfiles/virtual_attributes_50.gemfile
-- gemfiles/virtual_attributes_51.gemfile
-#- gemfiles/virtual_attributes_52.gemfile
-#- gemfiles/virtual_attributes_60.gemfile
+- gemfiles/gemfile_50.gemfile
```
Comment (NickLaMuro) [inline .travis.yml]: No where to put this, but there is a typo in your commit message:

> So it is easier ff they all are basically the same with the same file
> names.

`s/ff/if/`

File: .travis.yml (hunk 1 above)
Comment (kbrock) [reply .travis.yml]: sigh... thanks

Comment (kbrock) [reply .travis.yml]: well, feels like this has been in the works since 2.5.3 was the latest

Comment (NickLaMuro) [reply .travis.yml]: @kbrock I was commenting on the fact that your commit message said "test against latest versions of gems", but you are also updating the versions of ruby you are testing against.

Again, just trolling.

**Pull request merged**